modules/@apostrophecms/piece-page-type/index.js

```diff
@@ -38,6 +38,8 @@ module.exports = {
     self.piecesFilters = self.options.piecesFilters || [];
 
     self.enableAddUrlsToPieces();
+
+    self.originalChooseParentPage = self.chooseParentPage;
```

Comment:
At this stage of initialization subclasses haven't extended or overridden anything yet, so we can make a note of the original method for comparison later